yarn/README.md

```diff
@@ -10,6 +10,9 @@ This check collects metrics from your YARN ResourceManager, including (but not l
 * Per-application metrics (e.g. app progress, elapsed running time, running containers, memory use, etc.)
 * Node metrics (e.g. available vCores, time of last health update, etc/)
 
+### Deprecation notice
+`yarn.apps.<METRIC>` metrics have been deprecated in favour of `yarn.apps.<METRIC>_gauge` metrics. This is because `yarn.apps` metrics
```

Comment:
```suggestion
`yarn.apps.<METRIC>` metrics have been deprecated in favor of `yarn.apps.<METRIC>_gauge` metrics, because `yarn.apps` metrics are incorrectly reported as a `RATE` instead of a `GAUGE`.
```